pages/c/_cluster/apps/charts/index.vue

```diff
@@ -311,7 +399,18 @@ export default {
           {{ t('catalog.charts.header') }}
         </h1>
       </div>
+      <div class="actions-container">
+        <ButtonGroup
+          v-model="chartMode"
+          :options="chartOptions"
+        />
+      </div>
     </header>
+    <Carousel
```

Comment:
In the requirement is requested to do not display it also in case of no slides. Let's remove the logic from the condition and create a computed variable with
```
showCarousel() {
  return this.chartMode === 'featured' && this.sliders.length;
}

Reply:
Fixed